Sources_v3/StreamChat/ChatClient.swift

```diff
@@ -274,6 +274,7 @@ public class _ChatClient<ExtraData: ExtraDataTypes> {
         self.eventWorkerBuilders = eventWorkerBuilders
         
         createBackgroundWorkers()
+        createWebSocketClient()
```

Comment:
What's wrong with having the WS client initialized lazily?

Reply:
it is (was) initialized lazily, but that meant it was initialized in `createBackgroundWorkers` when it was passed to the workers. Tests assume this behavior (that wsclient is ready after `ChatClient.init`)
After I removed the dependency worker -> wsclient, tests failed because of that